### DISCUSSION TITLE
Add sound effects for some adventure spells

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [ihhub/fheroes2#7602] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Some adventure spells have a sound effect when being cast, like dimension door and town portal.

Though some spells like Set Guardians, Haunt and view spells have no sound effect. A sound effect isn't only nice and flavour to the game is also an audible confirmation to the player that the spell is cast. It feels as there is an sound effect missing when casting some spells, even though the OG doesn't have any effects.

Reusing som sounds like the Summon Elemental sounds for the Set Guardian spell, might be an option? The view spells could use the same sound as their effects are similar.

**Comment by ihhub** [comment]
I am moving this issue into the discussion section as it is a very broad topic and requires coordination of developers and SFX artists.

**Comment by Branikolog** [comment]
I personally believe, using summoning sound is a nice idea for "set guardian" spell.
As for haunt, I think we need a completely new sound. (which could also be used for battles, when ghost stack attacks and rises their amount)
